codeforboston/maple
# Bootstrap Migration for Bill Details

# Summary

Bootstrap Migration for Bill Details (for #1445 ). This includes the Sponsors, Committee, and (currently disabled) LobbyingTable components.

I moved these components to use the Bootstrap Card styling currently used by the Testimonies section - there are some slight differences between the old Sponsors style and the card style vis-a-vis border radius and header font size, but they're now at least consistent for all sections. 

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.
- [na] I've made pages responsive and look good on mobile.

# Screenshots
Old Sponsors and Committee: 
<img width="1189" alt="Screenshot 2024-03-13 at 2 05 37 AM" src="https://github.com/codeforboston/maple/assets/2694761/951c8956-bae0-4309-832e-b85698d399e5">

New Sponsors and Committee:
<img width="1206" alt="Screenshot 2024-03-13 at 2 05 29 AM" src="https://github.com/codeforboston/maple/assets/2694761/ba0855ec-dabe-4933-8c73-b72055637606">

LobbyingTable:
<img width="850" alt="Screenshot 2024-03-13 at 12 11 24 AM" src="https://github.com/codeforboston/maple/assets/2694761/887fb15c-b1f2-4428-9c9d-455368eaad19">


# Known issues

N/A

# Steps to test/reproduce

1. Go to a specific bill page (preferably, one associated with a committee and several cosponsors)
1. See that the components render as expected
